core/scripts/local/CreateMarginToken.js

```diff
@@ -1,16 +1,13 @@
 const TokenizedDerivativeCreator = artifacts.require("TokenizedDerivativeCreator");
 const AddressWhitelist = artifacts.require("AddressWhitelist");
-const ERC20MintableData = require("@openzeppelin/contracts/build/contracts/ERC20Mintable.json");
-const truffleContract = require("@truffle/contract");
-const ERC20Mintable = truffleContract(ERC20MintableData);
-ERC20Mintable.setProvider(web3.currentProvider);
+const ERC20Mintable = artifacts.require("Token");
```

Comment:
Out of curiosity, why call it `ERC20Mintable` instead of `Token`? Or maybe we should rename `Token` to something better haha.

Reply:
Yeah technically I should replace all instances of "ERC20Mintable" with "Token" but "Token" doesn't have a good name itself because its both mintable and burnable.

For consistency, I'll rename everything to Token